content/faq/lbry-privacy.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How does LBRY handle Privacy, and what data do you collect??
+category: other
+---
+## Is it possible to publish content anonymously? How is anonymity accomplished?
+
+LBRY does not log user download activity. However, by using our software, you are not anonymous on the network by default, similar to the way most torrenting software works. There are ways you can be more anonymous by using a VPN or other technologies. Please also refer to our [FAQs on Acceptable Use Policy](https://lbry.io/faq/acceptable-use-policy) if you’re thinking about doing anything illegal, as we don’t tolerate that.
+
+## What are the privacy mechanisms? Can publishers be connected to real-world identifiers, such as IP addresses? If not, how do you authenticate users so they can maintain control of their content?
+
+Yes, publishers can be linked to real-world identifiers if they choose to provide this info. This part of our software is still a work in progress. Identities are more like pseudonyms currently, although we believe we can support fully verified identities in the future.
+
+## If I download something, is there data that could identify me?
+
+LBRY uses similar file sharing methods to BitTorrent. Just like torrents, if your IP is observable on the network, you can be located in the physical world.
+
+## Is LBRY resistant to countries blocking content such as China or Iran?
+
+The answer to this is somewhat unclear. It is possible for  sophisticated enough software to detect LBRY and filter out its traffic.  Although if this happened, we could also evolve our technology, turning it into a fun game of cat-and-mouse. Currently, a lot of the traffic appears as random, encrypted UDP requests.
+
+## What information do you retain that could be subpoeaned?
+
+The LBRY Desktop application sends some analytics and diagnostic data that is discarded after several months. Technically, this data is subpoenable and could potentially be used to prove certain activities. However, this can be toggled off in the Settings under “Share Diagnostic Data.”
```

Comment:
We don't discard data right now, and not sure if we can certain data for rewards tracking purposes. Access data probably can be.